packages/labs/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@blueprintjs/labs",
+  "version": "0.0.0",
+  "description": "Incubator for unstable and in-development Blueprint components",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "style": "dist/blueprint-labs.css",
+  "unpkg": "dist/labs.bundle.js",
+  "dependencies": {
+    "@blueprintjs/core": "^1.18.0",
```

Comment:
nope, peer dependencies are annoying in general and should be avoided. there's no harm to having multiple versions of blueprint in your NPM tree, and everyone should be setting semver ranges appropriately so that they rarely get into that situation anyway. since we ship a single global CSS file, it's pretty hard to run into conflicts within a single app (your bundler will only pick up `node_modules/@blueprinjs/core/dist/blueprint.css`)